### PR TITLE
Ada2 19 working

### PR DESCRIPTION
**Pull request by imortimore**
had to switch to a new branch due to a git being a git and having issues with syncing should be all up to date as i lost the fixes we made during a call but i went through and re fixed so should be the same.

### CODE REVIEW

File: AgentDiscoveries-Backend/src/main/java/org/softwire/training/models/User.java [hunk 1]
```diff
@@ -51,5 +53,11 @@ public void setAgentId(Integer agentId) {
 
     public boolean isAdmin(){return admin;}
 
+    public boolean isAgent(){return agent;}
+
+
+
```
Comment (RogVan) [inline AgentDiscoveries-Backend/src/main/java/org/softwire/training/models/User.java:59]: Only need one new line here

File: AgentDiscoveries-Backend/src/main/resources/db/migration/V7__add_isAgent_col.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+ALTER TABLE users ADD COLUMN agent BOOLEAN NOT NULL DEFAULT False;
+
+UPDATE users SET agent = True WHERE username = 'test_agent';
```
Comment (RogVan) [inline AgentDiscoveries-Backend/src/main/resources/db/migration/V7__add_isAgent_col.sql:3]: You'll need an empty new line at the end of this file

File: AgentDiscoveries-Frontend/app/src/components/nav.jsx [hunk 3]
```diff
@@ -107,6 +102,29 @@ export default class NavigationBar extends React.Component {
...
+    }
+
+
+
+
+
+
+
+
+
```
Comment (RogVan) [inline AgentDiscoveries-Frontend/app/src/components/nav.jsx:127]: Only need one new line here as well

**Review by Archieduck — APPROVED** [review]
Looks good, region report error stacks but i don't think this is important

File: AgentDiscoveries-Backend/src/main/java/org/softwire/training/models/User.java (hunk 1 above)
Comment (RogVan) [inline AgentDiscoveries-Backend/src/main/java/org/softwire/training/models/User.java:59]: Careful having too many blank lines, would generally only have 1 here between the getters / setters

File: AgentDiscoveries-Frontend/app/src/components/submit-forms/submit-location-report.jsx [hunk 4]
```diff
@@ -33,7 +34,21 @@ export default class LocationReportSubmit extends React.Component {
...
+    // }
+    // onLogInEvent(){
+    //     this.setState({
+    //         isLoggedIn: isLoggedIn(),
+    //         isAdmin: isAdmin(),
+    //         isAgent: isAgent(),
+
+    //     });
+
+
```
Comment (RogVan) [inline AgentDiscoveries-Frontend/app/src/components/submit-forms/submit-location-report.jsx:51]: Remember not to include any commented out code you don't need

File: README.md [hunk 5]
```diff
@@ -35,7 +35,7 @@ To start the application from the command line, run
 
 ## Checking the application has started
 
-The application homepage can be found at `http://localhost:8080`.
+The application homepage can be found at `http://localhost:8080`.java -jar AgentDiscoveries-Backend/target/agentdiscoveries-backend-1.0-SNAPSHOT.jar
```
Comment (RogVan) [inline README.md:38]: Did you mean to add this?

**Review by RogVan — COMMENTED** [review]
Sorry I couldn't get round to this before you merged in, feel free to not make these changes if you are busy doing something else, but just for feedback purposes:

It all looks really good and the code all makes sense.

Just a minor point: No need to have more than one blank line separating things out, I used to do it all the time as well but it can make the file harder to read if everything is spread out more than it needs to be

**Pull request merged**